1-exercises/A-array-find/exercise.js

```diff
@@ -16,8 +16,13 @@ let names = [
   "Karim",
   "Ahmed",
 ];
-
-let longNameThatStartsWithA = findLongNameThatStartsWithA(names);
+function getlongNameThatStartsWithA(name) {
+  return name > 7 && name.startsWith("A");
+}
+let longNameThatStartsWithA = names.find(getlongNameThatStartsWithA);
+// let longNameThatStartsWithA = names.find((element) => {
+//   return element.length > 7 && element.startsWith("A");
+// });
 
```

Comment:
I really like the "startsWith" method you used. Nice! but can you think of another way of checking the second condition without using any method?